src/Features/CSharp/Portable/ConvertAutoPropertyToFullProperty/CSharpConvertAutoPropertyToFullPropertyCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,191 @@
+                document,
+                existingAccessors.getAccessor,
+                getAccessorStatement,
+                generator,
+                cancellationToken).ConfigureAwait(false);
+
+            SyntaxNode newSetter = null;
+            if (existingAccessors.setAccessor != null)
+            {
+                var setAccessorStatement = generator.ExpressionStatement(generator.AssignmentStatement(
+                            generator.IdentifierName(fieldName),
+                            generator.IdentifierName("value")));
+                newSetter = await AddStatementsToAccessorAsync(
+                    document,
+                    existingAccessors.setAccessor,
+                    setAccessorStatement,
+                    generator,
+                    cancellationToken).ConfigureAwait(false);
+            }
+
+            return (newGetAccessor: newGetter, newSetAccessor: newSetter);
+        }
+
+        private (AccessorDeclarationSyntax getAccessor, AccessorDeclarationSyntax setAccessor) GetExistingAccessors(AccessorListSyntax accessorListSyntax)
+        {
+            AccessorDeclarationSyntax getter = null;
+            AccessorDeclarationSyntax setter = null;
+            foreach (var accessor in accessorListSyntax.Accessors)
+            {
+                if (accessor.Kind() == SyntaxKind.GetAccessorDeclaration)
+                {
+                    getter = accessor;
+                }
+                else if (accessor.Kind() == SyntaxKind.SetAccessorDeclaration)
+                {
+                    setter = accessor;
+                }
+            }
+
+            return (getAccessor: getter, setAccessor: setter);
+        }
+
+        private static void GetExistingAccessors(AccessorListSyntax accessorListSyntax, ref AccessorDeclarationSyntax getAccessor, ref AccessorDeclarationSyntax setAccessor)
+        {
+            foreach (var accessor in accessorListSyntax.Accessors)
+            {
+                if (accessor.Kind() == SyntaxKind.GetAccessorDeclaration)
+                {
+                    getAccessor = accessor;
+                }
+                else if (accessor.Kind() == SyntaxKind.SetAccessorDeclaration)
+                {
+                    setAccessor = accessor;
+                }
+            }
+        }
+
+        private async Task<SyntaxNode> AddStatementsToAccessorAsync(
+                Document document,
+                SyntaxNode accessor,
+                SyntaxNode statement,
+                SyntaxGenerator generator,
+                CancellationToken cancellationToken)
+        {
+            var newAccessor = UpdateAccessor(accessor, statement);
+            newAccessor = await ConvertToExpressionBodyIfDesiredAsync(
+                document,
+                newAccessor,
+                cancellationToken).ConfigureAwait(false);
+
+            return await Formatter.FormatAsync(newAccessor, document.Project.Solution.Workspace).ConfigureAwait(false);
```

Comment:
i would not format here.  Formatting hsould happen automatically.  If that's not happening, you should just add a Formatter.Annotation to these nodes before returning them.